src/Sarif.Driver/PathExtensions.cs

```diff
@@ -19,6 +20,8 @@ namespace Microsoft.CodeAnalysis.Sarif.Driver
     /// <summary>Path extension functions.</summary>
     public static class PathExtensions
     {
+        private static readonly Regex s_invalidFileNameCharRegex = new Regex($"[{Regex.Escape(new string(Path.GetInvalidFileNameChars()))}]", RegexOptions.Compiled | RegexOptions.CultureInvariant);
```

Comment:
>Path.GetInvalidFileNameChars() [](http://example.com/codeflow?start=105&length=30)

not an actual issue,
just to note that with current way we will possibly generate different name in Linux and Windows, 
and a name without any invalid char (win+Linux) would be 
working in both os, easier to share.

the other way of cause is to hard code the combination of them.
each has its pros and cons, just to call it out.